### ISSUE TITLE
Apply the aphrodisiac effect on players infected because of leavers

### ISSUE THREAD

**Issue by Socialdarwinist** [necropotame/teeworlds-infclass#175]
Then humans get a chance – now, games suddenly end when someone in a crowd becomes boomer because of a leaver.

**Comment by duralakun** [comment]
I disagree with this. What i experience is -> 1 infected leaves -> another player gets infected -> 1 infected leaves -> ano... and so on until there are no players left. Your change would encourage rage quitting (for the infected). wich then causes another player get infected anyways, so its even more unfair

**Comment by Socialdarwinist** [comment]
I just had experienced ragequitting and emptying of a server before writing the issue because a leaver caused a medic who was healing to become a boomer. This is reality, and your chain scenario is unlikely.

As an addition, I suggest that not only the new infected gets inflicted by aphrodisiac but can also not be killed the same time. This would mean a restoration of the conditions before the first infections: Everyone is on a distance from the others.
